src/internal-api.yml

```
@@ -210,6 +210,37 @@ paths:
               application/json:
                 schema:
                   $ref: data/result/find.yml
+    "/app-api/v1/data/colls/{coll}/insertMany":
+      post:
+        tags:
+          - Data
+        summary: Inserts many documents in database
+        operationId: insertManyDocuments
+        parameters:
+          - name: coll
+            in: path
+            description: The documents collection name
+            required: true
+            schema:
+              type: string
+        requestBody:
+          description: The documents to create
+          required: true
+          content:
+            application/json:
+              schema:
+                type: array
+                items: 
+                  type: object
+        responses:
+          '200':
+            description: Documents inserted
+            content:
+              application/json:
+                schema:
+                  type: array
+                  items: 
+                    $ref: 'data/document.yml'
```

As explained on Discord, you do not return the documents but an object:

```suggestion
                  type: object
                  properties:
                    insertedIds:
                      type: array
                      items: 
                        type: string
                  required:
                    - insertedIds
```